build.sbt

```diff
@@ -20,7 +20,7 @@ developers := List(
 resolvers += Resolver.sonatypeRepo("releases")
 libraryDependencies ++= List(
   "org.eclipse.jgit" % "org.eclipse.jgit" % "4.5.4.201711221230-r",
-  "com.geirsson" %% "coursier-small" % "1.0.0-M2",
+  "com.geirsson" %% "coursier-small" % "1.0.0-M4",
```

Comment:
Add a comment why we need this. GitHub repo link. I guess it's coursier 1.0.0-M4. Why not the latest version?

Reply:
Done, added a comment.